.travis.yml

```diff
@@ -29,7 +29,3 @@ addons:
     - git
     - libgnome-keyring-dev
     - fakeroot
-
```

Comment:
Any particular reason for removing this? This creates two CI builds for PRs open from branches of this repo